index.js

```diff
@@ -321,4 +321,31 @@ form.addEventListener('submit', (e) => {
     const errorMessage = document.querySelector('small');
     errorMessage.classList.remove('display-none');
   }
-});
\ No newline at end of file
+});
+
+// Local Data Storage
+const myLocalStorage = {
+  name: '',
+  email: '',
+  message: '',
+};
+
+form.addEventListener('change', () => {
+  if (myLocalStorage.name !== null) {
+    myLocalStorage.name = document.getElementById('name').value;
+  }
+  if (myLocalStorage.name !== null) {
+    myLocalStorage.email = document.getElementById('email').value;
+  }
+  if (myLocalStorage.name !== null) {
+    myLocalStorage.message = document.getElementById('message').value;
+  }
+
+  localStorage.setItem('storageString', JSON.stringify(myLocalStorage));
+});
+
+const storageObject = JSON.parse(localStorage.getItem('storageString'));
```

Comment:
[OPTIONAL]
- You can consider using the example below to populate the input field on every reload :+1: 
> - example code
```javascript
window.onload = () => {
  let savedFormData = localStorage.getItem('contactFormInput');
  savedFormData = JSON.parse(savedFormData);
  // Check if the form data object is found on localStorage
  if (savedFormData) {
  // populate inputs values if data was found
  // ex: nameInput.value = savedFormData.name
  }
};

```

Reply:
THank you, I just did that too, good idea!